plans/feat-2987-shortcut-accent-color.md

```
@@ -0,0 +1,105 @@
+# feat(icons): ショートカットにアクセントカラー（淡い地色）を持たせる
+
+Issue: #2987 / refs #2960 / 前提: #2986（PR #2988, merged）
+
+## 問題
+
+#2960 の実際の困りごとは「`podcasts` / `rss_feed` / `menu_book` のような**同系統の汎用グリフ**が
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)